src/transformers/models/auto/configuration_auto.py

```diff
@@ -334,7 +334,6 @@
         ("dialogpt", "DialoGPT"),
         ("distilbert", "DistilBERT"),
         ("dit", "DiT"),
-        ("donut", "Donut"),
```

Comment:
I thought this was necessary for the title of the doc page to be rendered correctly